src/cmd/input.go

```diff
@@ -11,14 +11,16 @@ import (
 var dataFile string
 
 func readInputConfig() {
+	viper.SetConfigType("yaml")
 	switch dataFile {
 	case ".":
+		// TODO: does this block ever actually ever run?
 		viper.SetConfigFile("./data.yaml")
 	case "-":
 		if isStdInFromTerminal() {
-			log.Info().Msg("Reading input directly from command line...")
+			// TODO: this can take up to half a second to output which interrupts the user's experience
```

Comment:
🤔 If this is working correctly users shouldn't see this unless they are manually typing in yaml formatted input. If that's not the case lmk and I can take a closer look

Reply:
To clarify, this happens to me only when manually typing in the command. (The check `isStdInFromTerminal()` is fine)

What happens is: I start typing, then a second later the message pops up, I lose track of what I've typed, then `CTRL+C` and re-run the command and start typing again. It's a UX annoyance